text/0000-oci-image-annotations.md

```diff
@@ -0,0 +1,48 @@
+# OCI Image Annotations on Buildpacks
+
+## Summary
+
+All Paketo Buildpacks must have at least the following OCI image annotations with values as defined by the [OCI Image Format Specification's Pre-Defined Annotation Keys](https://github.com/opencontainers/image-spec/blob/v1.1.0/annotations.md#pre-defined-annotation-keys):
+* `org.opencontainers.image.source`
+* `org.opencontainers.image.revision`
+* `org.opencontainers.image.title`
+* `org.opencontainers.image.version`
+
+The following additional annotation are recommended:
+* `org.opencontainers.image.url`
+* `org.opencontainers.image.description`
```

Comment:
according to the spec you sent, would `org.opencontainers.image.created`, `org.opencontainers.image.authors`, `org.opencontainers.image.documentation`, etc be required as well?

Reply:
I don't believe they would be required, but if they are available, they should be specified. For paketo, I don't see any of that information readily available in `buildpack.toml` or elsewhere, so I didn't include them in this RFC.

Reply:
How do you know which fields in the upstream spec you sent are required?

Reply:
I'm assuming that no fields are required, only that if they are present, they are required to meet the requirements for that field's value.

That's what I see in the wild, at least. I can't find a single image that specifies all of those fields, but I see a number that specify some.

Reply:
Ah ok. The wording of the RFC `All Paketo Buildpacks must have at least the following OCI image annotations` made me think that this was required to be OCI compliant which was why I was confused.

I think this RFC seems like a good idea, especially if we can pull from the buildpack.toml fields as you suggested